MIP/mip-n/README.md

```diff
@@ -0,0 +1,39 @@
+# MIP-n: Naming Conventions for Movement Protocol Design
+- **Description**: Naming conventions for actors in Movement protocol design, tests, and documentation.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-0](../MD/md-0)
+
+## Abstract
+
+This MIP outlines the naming conventions for actors in Movement protocol design, tests, and documentation. By standardizing the naming conventions, we aim to ensure that the Movement community can easily understand and navigate the Movement protocol ecosystem.
+
+## Motivation
+
+Movement protocols are complex systems that involve multiple actors, such as users, developers, and validators. By establishing naming conventions for these actors, we can create a common language that simplifies communication and collaboration within the Movement community. Standardized naming conventions also make it easier for new participants to understand the Movement protocol ecosystem and contribute effectively.
+
+## Specification
+
+### Generic Names
+- Alice: a generic user who interacts with a Movement protocol. The first user in a generic setup SHOULD be named Alice.
+- Bob: a generic user who interacts with a Movement protocol. The second user in a generic setup SHOULD be named Bob.
+- Charlie: a generic user who interacts with a Movement protocol. The third user in a generic setup SHOULD be named Charlie.
+
+### Specific Names
+- (Malicious) Richard: a user who acts maliciously in a Movement protocol. The first malicious user in a generic setup SHOULD be named Richard.
+    - An alternative My Boy Richard can also be used to refer to a user who acts unnecessarily outside of their own interests, perhaps for the greater good. This usage was introduced to avoid being too univocal; Malicious Richard originally and endearingly referred to an early team member at Movement Labs. When specifying My Boy Richard, the usage of Richard should be prefixed. 
```

Comment:
```suggestion
    - An alternative My Boy Richard can also be used to refer to a user who acts unnecessarily outside of their own interests, perhaps for the greater good. This usage was introduced to avoid being too univocal; Malicious Richard originally and endearingly referred to an early team member at Movement Labs. When specifying My Boy Richard, the usage of Richard should be prefixed. 
    - Should "My Boy Richard" not be sufficient in describing the will or intention of the actor, "My Boy Primata" may be used when wishing to express an actor in the system who is incredibly generous, often acting for the greater good. You might think of Robin Hood as an example of a "My Boy Primata" actor.
    ```